testplan/testing/py_test.py

```diff
@@ -302,6 +302,16 @@ def _build_pytest_args(self):
         return pytest_args
 
 
+class PyTestRuntimeInfo(testing.RuntimeInfo):
+    """
+    This class provides information about the state of the actual test run
+    that is accessible from the testcase through the environment as:
+    ``env.pytest_runtime_info``. Not used for now.
```

Comment:
I prefer not to adding code until they are really needed.

Reply:
removed from PyTest